### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by devstastyshko**
Update readme.md

### CODE REVIEW

**Review by devstastyshko — COMMENTED** [review]
Good

File: README.md [hunk 1]
```diff
@@ -15,11 +15,29 @@ A todo app with user authentication
...
 - pull the project using git
-
+```cmd
+   git clone https://github.com/CryceTruly/django-rest-api.git
+   ```
 - create   a virtualenv using virtualenv
+```cmd
+   python -m pip install --upgrade pip
+   pip install virtualenv
+   virtualenv venv_name
```
Comment (CryceTruly) [inline README.md:25]: Thanks, @toqp  for raising this PR, could we also add setup instructions for windows users?

Comment (devstastyshko) [reply README.md:25]: Yes. I'll do it today.

**Comment by devstastyshko** [comment]
Good?

**Review by CryceTruly — APPROVED** [review]
well done @toqp 
LGTM

**Pull request merged**